internal/config/azure.go

```diff
@@ -132,3 +133,50 @@ func (c AzureTrustedLaunch) EqualTo(other AttestationCfg) (bool, error) {
 	}
 	return c.Measurements.EqualTo(otherCfg.Measurements), nil
 }
+
+// DefaultForAzureTDX returns the default configuration for Azure TDX attestation.
+func DefaultForAzureTDX() *AzureTDX {
+	return &AzureTDX{
+		Measurements: measurements.DefaultsFor(cloudprovider.Azure, variant.AzureTDX{}),
+		// TODO: Set default values for version numbers once enabled.
+		QESVN:      0,
+		PCESVN:     0,
```

Comment:
```suggestion
		// TODO: Set default values for version numbers once enabled.
		QESVN:      math.MaxUint16,
		PCESVN:     math.MaxUint16,
```

nit: those are safer placeholders for a minimal version compare.

Reply:
Those are not placeholders anymore, removed the comment